guessNumber.js

```diff
@@ -0,0 +1,19 @@
+var rNumber = Math.floor(Math.random()* 10);
+
+function guessNumber() {
+    
+    for(let i = 0; i < 3; i++){
```

Comment:
Even if task is to work with specific values it's good to create options like:
const minNumber=1;
const maxNumber=10;
const numOfTrials=3;
and then use in code these variables everywhere.